addons/test_event_full/tests/test_event_security.py

```diff
@@ -142,6 +144,52 @@ def test_event_access_event_manager(self):
             })
             event_config.execute()
 
+    def test_event_access_event_question_and_event_answer(self):
+        """ Check that some user groups have access to questions and answers only if they are linked to at
+        least one published event. """
+        user_internal = mail_new_test_user(
+            self.env,
+            name='User Internal',
+            login='user internal',
+            email='user_internal@example.com',
+            groups='base.group_user',
+        )
```

Comment:
Ok, quick pong :ping_pong: so we can be done with that part ^^
maybe `user_internal_event` ? and login usually do not use space -> `user_internal_event` too (this is to avoid the generic user_internal that could be also used in other tests and crash as we do not want identical logins)

Finally, you can remove `self.admin_user` from the test, not really relevant.

Also, you still have two `has_access`, just use `read(...)` it should be good enough

Thanks for the updates!